openwisp_radius/base/models.py

```diff
@@ -1139,6 +1139,11 @@ def allowed_mobile_prefixes_list(self):
             mobile_prefixes = self.allowed_mobile_prefixes.split(',')
         return mobile_prefixes
 
+    def get_sms_verification_enabled(self):
```

Comment:
Since the field name is `sms_verification`, it will be better to rename this to `get_sms_verification`.

Reply:
I will make the requested changes.